write.js

```diff
@@ -12,17 +12,24 @@ class ChunkStoreWriteStream extends stream.Writable {
     if (!chunkLength) throw new Error('Second argument must be a chunk length')
 
     const zeroPadding = opts.zeroPadding === undefined ? false : opts.zeroPadding
-    this._blockstream = new BlockStream(chunkLength, { zeroPadding })
+    this._blockstream = new BlockStream(chunkLength, { ...opts, zeroPadding })
     this._outstandingPuts = 0
+    this._storeMaxOutstandingPuts = opts.storeMaxOutstandingPuts || 16
 
     let index = 0
     const onData = chunk => {
       if (this.destroyed) return
 
       this._outstandingPuts += 1
+      if (this._outstandingPuts >= this._storeMaxOutstandingPuts) {
+        this._blockstream.pause()
```

Comment:
@feross can we pause torrents now?

Reply:
No, this doesn't have to do with pausing torrents. This PR was about adding backpressure to the seed method. So it won't try to read in data to the chunk store faster than it can handle.